lib/iris/analysis/calculus.py

```diff
@@ -458,16 +463,32 @@ def curl(i_cube, j_cube, k_cube=None, ignore=None):
         The k cube of the vector to operate on
     * ignore
         This argument is not used.
+
         .. deprecated:: 0.8
             The coordinates to ignore are determined automatically.
 
     Return (i_cmpt_curl_cube, j_cmpt_curl_cube, k_cmpt_curl_cube)
 
+    If the k-cube is not passed in then the 2-dimensional curl will
+    be calculated, yielding a single cube representing the vertical component
+    as the result.  If the k-cube is passed in, the 3-dimensional curl will
+    be calculated, returning 3 component cubes.
+
+    All cubes passed in must have the same data units, and those units
+    must be spatially-derived (i.e. m/s or km/h).
+
     The calculation of curl is dependent on the type of
-    :func:`iris.coord_systems.CoordSystem` in the cube:
+    :func:`iris.coord_systems.CoordSystem` in the cube.
```

Comment:
handy hint : you can reduce the link text to the final name (leaving out module path) with a tilde (\~).
E.G. :func:\`~iris.coord_systems.CoordSystem\`